docs/landing-zone-v2/README.md

```diff
@@ -633,13 +633,17 @@ To reacquire the resources you will need to redeploy a new instance and deploy t
 First delete the Rootsync deployment. This will prevent the resources from self-healing.
 
 ```shell
-kubectl delete root-sync landing-zone -n config-management-system
+kubectl delete RootSync root-sync -n config-management-system
 ```
 
 Now we can delete our KCC resources from the Config Controller instance.
 
 ```shell
-kubectl delete gcp --all
+kubectl delete gcp -n projects --all
```

Comment:
the `config-control` namespace also has resources

the sequence for deletion is important because of dependencies
for example, some IAM resources in `config-control`, `projects` and `hierarchy` assign permissions to GCP service accounts (ex: `logging-sa`) used to manage other resources (ex: log sinks in `logging` namespace, etc). If the permissions are removed before the resource is deleted, it'll get in a failed state.

the following sequence may help to avoid conflicts (hasn't been tested):
```bash
kubectl delete gcp -n networking --all
kubectl delete gcp -n logging --all
kubectl delete gcp -n policies --all
kubectl delete gcp -n projects --all
kubectl delete gcp -n hierarchy --all
kubectl delete gcp -n config-control --all
```

Reply:
Done.

Reply:
Just figured out a better approach as shown below.
`kpt live destroy <solution_folder_path>`
We just need to destroy each of deployed solutions in a reverse order one by one, right? @fmichaelobrien and @alaincormier-ssc